reactive/kotlinx-coroutines-reactive/src/Await.kt

```diff
@@ -81,6 +82,14 @@ public suspend fun <T> Publisher<T>.awaitSingle(): T = awaitOne(Mode.SINGLE)
 
 // ------------------------ private ------------------------
 
+// ContextInjector service is implemented in `kotlinx-coroutines-reactor` module only.
+// If `kotlinx-coroutines-reactor` module is not included, the list is empty.
+private val contextInjectors: List<ContextInjector> =
```

Comment:
It makes sense to invoke `first` then to have an instance instead of a foldable list.
Note that `first` should be used, not `single`, otherwise some third-party code can make library unusable.

Reply:
`first` will not save us from 3rd party code, as there will be no guarantee on the order. This way it is, at least, extensible (theoretically). I've changed the type to array to get a little bit of extra performance (removing additional wrapping layer)